app/javascript/hyacinth_ui_v1/components/controlled_vocabularies/terms/Terms.js

```diff
@@ -36,6 +26,14 @@ export default class Terms extends React.PureComponent {
           )}
         />
 
+        <ProtectedRoute
+          path="/controlled_vocabularies/:stringKey/terms/:uri"
+          component={TermShow}
+          requiredAbility={params => (
+            { action: 'read', subject: 'Term', id: params.id }
```

Comment:
Will this work?  I don't think we'll be assigning permissions for specific Term ids.

Reply:
It will work. It is being very specific about the permissions its checking and it might be unnecessary, but it future proofs us in case we start limiting access by identifier.

I also wonder if we should open up the terms/vocabulary show pages to all users. But that might be a HUG question.

Reply:
As per our conversation about this, I've ticketed removal of this required ability check.

I think opening up terms/vocabulary show pages is a great idea.